src/Interpreters/InterpreterDropQuery.cpp

```diff
@@ -15,7 +15,10 @@
 #include <Storages/StorageMaterializedView.h>
 #include <Common/escapeForFileName.h>
 #include <Common/quoteString.h>
+#include <Common/typeid_cast.h>
 #include <Common/thread_local_rng.h>
+#include <Common/likePatternToRegexp.h>
+#include <re2/re2.h> 
```

Comment:
Could we use this instead?
```suggestion
#include <Common/re2.h> 
```
This header file seems to be a wrapper around `<re2/re2.h>` with some custom clang diagnostic overrides and is being used in other places.